src/app/components/profile/profile.component.ts

```diff
@@ -151,20 +151,23 @@ export class ProfileComponent implements OnInit {
         {
           isRequired: true,
           label: 'first_name',
+          name: 'update_first_name',
```

Comment:
If you want, now we should have `interfaces`, that is,
form: YourDefinedInterface[ ] = [Filling values here, as done currently]